ui/pages/routes/routes.component.js

```diff
@@ -928,6 +929,9 @@ export default class Routes extends Component {
         {isNetworkMenuOpen ? (
           <NetworkListMenu onClose={() => toggleNetworkMenu()} />
         ) : null}
+        {process.env.ENABLE_NETWORK_UI_REDESIGN ? (
+          <NetworkConfirmationPopover />
```

Comment:
Can this be added in `network-list-menu.js` instead of `Routes`?

Reply:
The issue with incorporating this into network-list-menu.js lies in its nature as a modal component. For the functionality in question, utilizing a popover necessitates it being part of a standard component that is associated with a specific route, rather than being embedded within a modal.